sanic/request.py

```diff
@@ -79,7 +79,7 @@ def token(self):
         :return: token related to request
         """
         auth_header = self.headers.get('Authorization')
-        if 'Token ' in auth_header:
+        if auth_header is not None and 'Token ' in auth_header:
```

Comment:
I'd prefer it being None, as it's being passed in the else branch. One thing I'd like to check is `request.token is not None`, as it's super clean to read.

The other way to make it return `None`, yet use `''` would be to return an explicit none in the else branch. Not sure how I feel about the readability of the latter, but you guys are the core devs, gonna do it the way you choose.